evoting/lib/transaction.go

```diff
@@ -99,8 +99,40 @@ func (t *Transaction) Digest(s *skipchain.Service, genesis skipchain.SkipBlockID
 // Verify checks that the corresponding transaction is valid before storing it.
 func (t *Transaction) Verify(genesis skipchain.SkipBlockID, s *skipchain.Service) error {
 	digest := t.Digest(s, genesis)
-	db := s.GetDB()
 	if t.Master != nil {
+		// Find the current master in order to compare against it.
+		m, err := GetMaster(s, genesis)
```

Comment:
The `s *skipchain.Service` is the skipchain that contains the master blocks, right? But the transaction is, to my understanding, something on the election skipchain. Don't you want to get, not the latest master, but the master that is just before the Election/transaction block?

Reply:
(We figured out this misunderstanding in person.)